io-engine/src/bdev/nexus/nexus_io.rs

```diff
@@ -185,6 +191,12 @@ impl<'n> NexusBio<'n> {
 
     /// TODO
     pub(super) fn submit_request(mut self) {
+        if self.channel().is_suspended() {
```

Comment:
I wonder if we need to take a different approach for when etcd is down for some time, and initiator IO timeout, presumably we need to abort IOs that we've actually "paused" so we don't fire them up when we switch back to "normal mode" ?

Reply:
Yes, it makes sense.